web/Views/Admin/TenantMatchList.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@model TenantMatchListModel
+
+@{
+    ViewData["Title"] = "Tenant List";
+}
+
+<h2>Tenant Matches for @Model.CurrentProperty.Address.AddressLine1 @Model.CurrentProperty.Address.AddressLine2 @Model.CurrentProperty.Address.AddressLine3</h2>
+
+<table class="table">
+    <thead>
+    <tr>
+        <th scope="col">Name</th>
+        <th scope="col">Email</th>
+        <th scope="col">Phone</th>
+        <th scope="col">Postcode</th>
+        <th scope="col">Type</th>
+        <th scope="col">Has Pet?</th>
+        <th scope="col">In ETT?</th>
+        <th scope="col">On Universal Credit?</th>
+        <th scope="col">On Housing Benefits?</th>
+        <th scope="col">Over 35?</th>
+        <th scope="col">Match</th>
+    </tr>
+    </thead>
+    <tbody>
+    @foreach (var tenant in Model.TenantList!)
+    {
+        <tr>
+            <td>@tenant.Name</td>
+            <td>@tenant.Email</td>
+            <td>@tenant.Phone</td>
+            <td>@tenant.Postcode</td>
+            <td>@tenant.Type</td>
+            <td>@tenant.HasPet</td>
+            <td>@tenant.ETT</td>
+            <td>@tenant.UniversalCredit</td>
+            <td>@tenant.HousingBenefits</td>
+            <td>@tenant.Over35</td>
+            <td>
+                @using (Html.BeginForm("SendMatchLinkEmail", "Admin", new { propertyLink = Model.CurrentProperty.Description, /*TODO: change this from description to link*/ 
+                                                                                                        tenantEmail = tenant.Email, 
+                                                                                                        currentPropertyId = Model.CurrentProperty.PropertyId }, FormMethod.Post))
```

Comment:
indentation has gone very far here. I think we can add some more new lines in here to make it tidier :)
```suggestion@using (Html.BeginForm("SendMatchLinkEmail", "Admin", new
                {
                    propertyLink = Model.CurrentProperty.Description, /*TODO: change this from description to link*/ 
                    tenantEmail = tenant.Email,
                    currentPropertyId = Model.CurrentProperty.PropertyId
                }, FormMethod.Post))
```